cosmos_rl/policy/model/qwen3_moe/weight_mapper.py

```diff
@@ -109,7 +128,7 @@ def rollout_prepare_recv(
                 vllm_weight_inplace_view_map[param_name_hf] = param
                 group_keys.append((param_name_hf, param.ndim))
             recv_key_n_rank_list.append(group_keys)
-        return vllm_weight_inplace_view_map, recv_key_n_rank_list
+        return vllm_weight_inplace_view_map, recv_key_n_rank_list, vllm_full_weight_map
```

Comment:
would this return type change break existing out of source tree weight mapper? cc @aazzolini